endpoints/openrtb2/amp_auction_test.go

```diff
@@ -122,357 +122,291 @@ func TestAMPPageInfo(t *testing.T) {
 	assert.Equal(t, "test.somepage.co.uk", exchange.lastRequest.Site.Domain)
 }
 
-func TestConsentThroughEndpoint(t *testing.T) {
```

Comment:
The existing GDPR consent tests were difficult to work with. I converted them into a table driven test and added CCPA tests as well.